src/test/regress/bin/normalize.sed

```diff
@@ -65,6 +65,14 @@ s/"(ref_table_[0-9]_|ref_table_[0-9]_value_fkey_)[0-9]+"/"\1xxxxxxx"/g
 /^LINE [0-9]+:.*$/d
 /^ *\^$/d
 
+# PG 11 and PG 12 differs in terms of CTE inlining
```

Comment:
let's add `PG_VERSION_NUM < 120000` somewhere in this comment. That way we know we can remove this when we stop supporting PG 11.

Reply:
This is already merged into master via your PR, so skipping